scripts/build_android.sh

```diff
@@ -15,22 +15,24 @@
 #         brew install automake
 #         brew install libtool
 # (2) You will need to have android ndk installed. The current script assumes
-#     that you are installing it under /opt. If not, you will need to adjust
-#     accordingly.
+#     that you set ANDROID_NDK to the location of ndk.
 # (3) The toolchain and the build target platform can be specified with the
 #     cmake arguments below. For more details, check out android-cmake's doc.
 
 CAFFE2_ROOT="$( cd "$(dirname "$0")"/.. ; pwd -P)"
 echo "Caffe2 codebase root is: $CAFFE2_ROOT"
-# Now, actually build the android target. Let's find the most recent version
-# of android ndk under /opt/android_ndk.
-NDK_VERSION="$(ls -1 /opt/android_ndk/ | sort | tail -1)"
-if [ -z "$NDK_VERSION" ]; then
-    echo "Cannot find ndk: did you install it under /opt/android_ndk?"
+
+if [ -z "$ANDROID_NDK" ]; then
+    echo "Did you set ANDROID_NDK variable?"
```

Comment:
A default path is already added in `.travis/build.sh` so that it won't break Travis-ci task. 

For the script itself, it's better to prompt users to set the NDK path if they didn't.  It's not a good idea to specify a default NDK location since different users may have different configurations.